src/EDLog/EDLog.ts

```diff
@@ -406,3 +409,19 @@ export class EDLog extends EventEmitter {
         return <GameEvents[K]> this.backlog.filter(ev => ev.event === realEvent);
     }
 }
+
+export interface IBacklogOptions {
+    /**
+     * If true the method will return an array of event logs, otherwise empty.
+     * 
+     * @memberof IBacklogOptions
```

Comment:
This is also redundant, it's already clearly defined what construct this member belongs to.